Snakefile

```diff
@@ -1,3 +1,7 @@
+rule target:
+    input: 
+        "01-getting-started/getting-started.html" 
+
 rule render_citations:
     input:
         "01-getting-started/getting-started.Rmd"
```

Comment:
This is where the `output:` comes in. It allows snakemake to trace backward to figure out which rules need to be run. Right now, you probably already have `getting-started.html` so when you run `snakemake` it says there is nothing to do. What you want to have happen is that when you delete that file (or modify the .Rmd) the steps to generate the file are rerun. To do that, you need things linked up so every input in the target is the same as a an output from some rule (and all the inputs from that are outputs from somewhere, or separately provided files).

```suggestion
        "01-getting-started/getting-started.Rmd"
    output:
        ""01-getting-started/getting-started.html"
```

Reply:
Implemented in https://github.com/AlexsLemonade/refinebio-examples/pull/150/commits/096f52610d31757e65513c3d6af8852506fad524